### DISCUSSION TITLE
Not able to run a simple observable hello world example

### DISCUSSION THREAD

**Discussion by aborruso** [neocarto/bertin#113] [Q&A] [answered]
Hi,
I have uploaded Italian regions geojson and then I have inserti this code

```
Reg01012023_g_WGS84 = FileAttachment("regioni_italia@1.geojson").json()
```

Then I have used this map code

```
simplemap = bertin.draw({
  params: { background: "#bde1f0", margin: 10 },
  layers: [
    {
      type: "layer",
      geojson: Reg01012023_g_WGS84,
      fill: "#c40001",
      stroke: "#e8d203",
      strokeWidth: 1.5
    }
  ]
})
```

but I do not have Italy. What's wrong in [my code](https://observablehq.com/d/fa939367237d9243#simplemap)?

Thank you for this great tool!!

![image](https://github.com/neocarto/bertin/assets/30607/567dd2a0-32ce-497b-b12f-8d8d1e5f8ba9)

![image](https://github.com/neocarto/bertin/assets/30607/a3b6af71-234a-42fc-92c6-4a546a1fdbfd)

**Comment by Fil** [comment, marked as answer]
You have to fix the geometries, for example with https://observablehq.com/@fil/rewind:

```
import {rewind} from "@fil/rewind"
Reg01012023_g_WGS84 = FileAttachment("regioni_italia@1.geojson").json().then(rewind)
```

**Reply by aborruso** [reply]
Thank you very much

**Comment by neocarto** [comment]
You can also use rewind directly within bertin.

    {
      type: "layer",
      geojson: Reg01012023_g_WGS84,
     rewind: true
  }

**Reply by aborruso** [reply]
Wow, thank you

**Reply by aborruso** [reply]
> You can also use rewind directly within bertin.

I have applied it here https://observablehq.com/d/fa939367237d9243#simplemap

Very useful
